torch/csrc/autograd/variable.cpp

```diff
@@ -789,7 +789,6 @@ void handle_view_on_rebase(
           " allow the output views to be modified inplace. You should replace the inplace operation by an"
           " out-of-place one.");
     } else if (creation_meta == CreationMeta::NO_GRAD_MODE) {
-      TORCH_INTERNAL_ASSERT(!grad_fn);
```

Comment:
Actually could we update the error message to be even more precise on this case: you did a differentiable view on a view created in a no_grad block and are now modifying inplace.

Reply:
Good idea, updated.